app/services/decisions/income_decision_tree.rb

```diff
@@ -36,12 +35,25 @@ def after_employment_status
       end
     end
 
+    def after_income_before_tax
+      if income_below_threshold?
+        edit(:client_owns_property)
```

Comment:
First diamond is the freezing order one, not the property one (property is also CC only). Not a huge problem, as we will come to checking the order later, but could be worth putting a TODO in to put the right page in when we have it